src/Microsoft.DocAsCode.MarkdigEngine.Extensions/TripleColon/ImageExtension.cs

```diff
@@ -94,23 +94,30 @@ public bool TryProcessAttributes(IDictionary<string, string> attributes, out Htm
             {
                 htmlAttributes.AddProperty("alt", alt);
             }
-            var id = GetHtmlId(block);
+            var id = GetHtmlId(markdownObject);
             if(type == "complex") htmlAttributes.AddProperty("aria-describedby", id);
 
             RenderDelegate = (renderer, obj) =>
             {
+                dynamic blockOrInline;
```

Comment:
Is there a way to not using `dynamic` here? `dynamic` is likely to hurt the performance of this method: https://stackoverflow.com/questions/7478387/how-does-having-a-dynamic-variable-affect-performance

Reply:
Does it look better now @yufeih ?

Reply:
Yep, much better. There are two tests failing though.

Reply:
@yufeih - It looks like those tests are passing now. I noticed the regression diff tests (is that what you call them?) were not passing so I downloaded one of the diff files, and it had lots of this:
![image](https://user-images.githubusercontent.com/7014528/84409538-ce3bf200-abca-11ea-9847-848177e31363.png)

That is now what the image output should be, so we should be good to merge this, yes?

Reply:
yes